plugins/modules/win_nssm.py

```diff
@@ -100,6 +100,10 @@
     type: str
     choices: [ auto, delayed, disabled, manual ]
     default: auto
+  app_environment:
+    description:
+      - Key/Value pairs which will be added to the environment of the service application.
+    type: dict
```

Comment:
Should have `version_added: 1.2.0` to denote this was added in that collection version.

Reply:
Done